cpp/log/frontend_signer.cc

```diff
@@ -70,13 +72,7 @@ FrontendSigner::SubmitResult FrontendSigner::QueueEntry(
     *sct = new_logged.sct();
   }
 
-  if (status.CanonicalCode() == util::error::ALREADY_EXISTS) {
-    return DUPLICATE;
-  } else {
-    // TODO(alcutter): really need to propagate failures back up!
-    CHECK(status.ok());
```

Comment:
And, more importantly, this just gets sent back to the client, rather than setting things on fire? :smiley: